scripts/check-newsfragment

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+#
+# A script which checks that an appropriate news file has been added on this
+# branch.
+
+
+echo -e "+++ \033[32mChecking newsfragment\033[m"
+
+set -e
+
+# make sure that origin/develop is up to date
+git remote set-branches --add origin develop
+git fetch -q origin develop
+
+pr="$PULL_REQUEST_NUMBER"
+
+# Print a link to the contributing guide if the user makes a mistake
+CONTRIBUTING_GUIDE_TEXT="!! Please see the contributing guide for help writing your changelog entry:
+https://github.com/matrix-org/matrix-appservice-irc/blob/develop/CONTRIBUTING.md#making-pull-requests"
```

Comment:
This needs updating!

Reply:
This was copied from [matrix-appservice-bridge](https://github.com/matrix-org/matrix-appservice-bridge/blob/develop/scripts/check-newsfragment#L19), so it needs updating too...!